tests/neg-custom-args/no-experimental/experimental.scala

```diff
@@ -0,0 +1,25 @@
+class Test0 {
+  import language.experimental.namedTypeArguments // error
```

Comment:
We should also have a test with `import scala.language.experimental`

Reply:
You mean `import scala.language.experimental.namedTypeArguments`, with the prefix `scala.`?